cygnus-twitter/spec/SPECS/cygnus-twitter.spec

```diff
@@ -99,6 +99,8 @@ rm -rf $RPM_BUILD_ROOT
 %{_project_install_dir}
 
 %changelog
+* Tue Jan 31 2016 Francisco Romero <frnacisco.romerobueno@telefonica.com> 1.7.0
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1405/commits/0d66abddcd20be9a33bc46ba8688a7a0dd94805d